samples/unit-tests/stock-tools/bindings/demo.js

```diff
@@ -398,3 +398,65 @@ QUnit.test('Bindings general tests', function (assert) {
         qunitContainer.style.display = 'block';
     }
 });
+
+QUnit.test('Stock Tools annotations\' positions with yAxis.top (#15075)', assert => {
+    const chart = Highcharts.stockChart('container', {
+            chart: {
+                width: 800
+            },
+            yAxis: {
+                top: '40%',
+                height: '60%'
+            },
+            series: [{
+                data: [4, 2, 5, 6, 4]
+            }]
+        }),
+        series = chart.series[0],
+        yAxis = series.yAxis;
+
+    let point = series.points[1];
+
+    // Add vertical counter annotation near to the second point
+    chart.navigationBindings.options.bindings.verticalLabel.start.call(
+        chart.navigationBindings,
+        {
+            chartX: point.plotX,
+            chartY: point.plotY + yAxis.top
+        }
+    );
+
+    let annotationBBox = chart.annotations[0].graphic.getBBox();
+
+    assert.strictEqual(
+        annotationBBox.y + annotationBBox.height + 10,
```

Comment:
Could you please explain from where this `10` came?
Is that some kind of predefined offset?

Reply:
I thought that the assert description ```'Annotation\'s element should be placed 10px from the second point.'``` will be enough comment. Some annotations are placed 10px from a point. I haven't searched the code where exactly those 10 pixels are set, but it's a standard position. I can add an additional comment.

Reply:
The test description is pretty self-explanatory and checks that condition perfectly.
I was curious if there is an option to replace `10` with something like (it's just a guess) `annotation.offset`

Reply:
I'll check

Reply:
You were close, thanks :) `typeOptions.yOffset`
https://github.com/highcharts/highcharts/blob/695395b8bd374a70e14a7863cc3dd34b8a0e1acb/ts/Extensions/Annotations/Types/VerticalLine.ts#L189